app/flows/student_finance_calculator_flow/outcomes/outcome_uk_all_students.erb

```diff
@@ -15,15 +15,7 @@
     - a Graduate Certificate or Graduate Diploma
   <% end %>
 
-  ##Student finance
-
-  You could get per year:
-
-  - <%= format_money(calculator.tuition_fee_amount, pounds_only: true) %> [Tuition Fee Loan](/student-finance/parttime-students)
-
-  <% if calculator.maintenance_loan_amount > 0 %>
-    - <%= format_money(calculator.maintenance_loan_amount, pounds_only: true) %> [Maintenance Loan](/student-finance/parttime-students)
-  <% end %>
+  <%= render partial: 'tuition_maintenance_summary', locals: { tuition_fee_amount: calculator.tuition_fee_amount, maintenance_loan_amount: calculator.maintenance_loan_amount, maintenance_grant_amount: calculator.maintenance_grant_amount, start_date: calculator.course_start, max_loan_amount: calculator.max_loan_amount, tuition_fee_url: "/student-finance/parttime-students", maintenance_loan_url: "/student-finance/parttime-students" } %>
```

Comment:
TODO: another commit to split these parameters onto separate lines, now that there are so many of them. (Do this for all outcomes).